src/utils/app_utils.py

```
@@ -51,247 +59,180 @@ def get_ip_address():
-            )
-        except Exception:
-            max_upload_bytes = 10 * 1024 * 1024
+        # Read raw bytes once so this works with both FileStorage and test fakes.
+        content = file.read()
+        if content is None:
+            raise RuntimeError("Empty upload content")
+
+        max_upload_bytes_env = os.getenv("MAX_UPLOAD_BYTES")
+        max_upload_bytes = int(max_upload_bytes_env) if max_upload_bytes_env else 10 * 1024 * 1024
+        if len(content) > max_upload_bytes:
+            raise RuntimeError(f"Uploaded file exceeds size limit of {max_upload_bytes} bytes")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)